### PR TITLE
Capital letters in const, remove default data in class Car

### PR DESCRIPTION
**Pull request by GrzegorzZajac000**
Capital letters in const, remove default data in class Car

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1148,9 +1148,9 @@ car.save();
 ```javascript
 class Car {
   constructor() {
-    this.make = 'Honda';
-    this.model = 'Accord';
-    this.color = 'white';
+    this.make = '';
```
Comment (ryanmcdermott) [inline README.md]: Why were these deleted?

Comment (GrzegorzZajac000) [inline README.md]: Because it has no sense. For example you add new Honda to user and then you set it to Ford?

Comment (ryanmcdermott) [inline README.md]: You're right. Let's pass in the `make`, `model`, and `color` into the constructor.

**Comment by ryanmcdermott** [comment]
Thanks for your help!
